tensorpack/dataflow/format.py

```diff
@@ -103,6 +118,8 @@ def get_data(self):
                 if k != '__keys__':
                     yield [k, v]
         else:
+            if not hasattr(self, "rng"):
+                self.reset_state()
```

Comment:
Ok. But I want to test it again including the Prefetching(ZMQ) before pushing new code. So it would be Monday at the earliest or Sunday evening.